script/FullDeployer.s.sol

```diff
@@ -51,16 +55,16 @@ contract FullDeployer is HubDeployer, ExtendedSpokeDeployer {
             version: vm.envOr("VERSION", bytes32(0))
         });
 
-        // Use the regular deployment functions - they now use CreateX internally
-        deployFull(input, msg.sender);
+        FullActionBatcher batcher = new FullActionBatcher();
```

Comment:
The batcher is constructed using CREATE1 because it's a temporal contract, just used for the permissions during the deployment phase.